operating/manage-user-access/configuring-user-authentication-on-rhacs.adoc

```diff
@@ -0,0 +1,34 @@
+:_content-type: ASSEMBLY
+[id="configuring-user-authentication-on-rhacs"]
+= Configuring user authentication on {product-title-short}
+include::modules/common-attributes.adoc[]
+:context: user-authentication
+
+toc::[]
+
+[role="_abstract"]
+
+Ensure that {rh-rhacs-first} is connected to atleast one Identity Provider (IDP) for user authentication.
+
+You can set up your own IDP right after provisioning using one of the guides under `Operating -> Managing user access`.
+After the IDP authenticates you, {product-title-short} maps your account to roles that determine your permissions and access scope.
+
+To authorize users, {product-title-short} relies on mapping rules that examine the values of specific claims like, `groups`, `email`, `userid` and `name` from either the user ID token or UserInfo endpoint response in OpenID Connect (OIDC).
+
+If these details are absent, the mapping cannot succeed and the user does not get access to the required resources.
+Therefore, you need to ensure that the claims required to authorize users from your IDP, for example, `groups`, are included in the authentication response of your IDP to enable successful mapping.
+
+For example, to connect an Azure Active Directory (AD) to {product-title-short} using OIDC, see xref:../../operating/manage-user-access/configuring-user-authentication-on-rhacs.adoc#connecting-azure-ad-to-rhacs-using-oidc_user-authentication[Connecting Azure AD to RHACS using OIDC].
+
+[id="connecting-azure-ad-to-rhacs-using-oidc_{context}"]
+== Connecting Azure AD to {product-title-short} using OIDC
```

Comment:
Here we say `using OIDC`, while below I see `for SAML applications` which are two different auth protocols.

Reply:
Updated title